pkgs/applications/science/logic/abc/default.nix

```diff
@@ -11,6 +11,9 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ readline ];
+  preBuild = ''
+    export buildFlags="CC=$CC CXX=$CXX LD=$LD"
```

Comment:
That patch is Darwin-specific? Why does Darwin need that variable, but other platforms don't?